sklearn/cluster/_agglomerative.py

```diff
@@ -818,6 +818,9 @@ class AgglomerativeClustering(ClusterMixin, BaseEstimator):
         `kneighbors_graph`. Default is ``None``, i.e, the
         hierarchical clustering algorithm is unstructured.
 
+        For an example of connectivity matrix using `kneighbors_graph`, see
```

Comment:
```suggestion
        For an example of connectivity matrix using :class:`~sklearn.neighbors.kneighbors_graph`, see
```

You might also need to wrap the line to comply with the line length limits.

Reply:
@adrinjalali Thanks for the suggestions, I've updated the docs